DigitalLearningSolutions.Web/Controllers/Support/ResourcesController.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers.Support
+{
+    using System.Threading.Tasks;
+    using DigitalLearningSolutions.Data.Enums;
+    using DigitalLearningSolutions.Data.Helpers;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Attributes;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using DigitalLearningSolutions.Web.ViewModels.Support.Resources;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.FeatureManagement;
+
+    [Route("/{dlsSubApplication}/Resources")]
+    [Authorize(Policy = CustomPolicies.UserCentreAdminOrFrameworksAdmin)]
+    [SetDlsSubApplication]
+    [SetSelectedTab(nameof(NavMenuTab.Support))]
+    public class ResourcesController : Controller
+    {
+        private readonly IConfiguration configuration;
+        private readonly IFeatureManager featureManager;
+        private readonly IResourcesService resourcesService;
+
+        public ResourcesController(
+            IFeatureManager featureManager,
+            IConfiguration configuration,
+            IResourcesService resourcesService
+        )
+        {
+            this.featureManager = featureManager;
+            this.configuration = configuration;
+            this.resourcesService = resourcesService;
+        }
+
+        public async Task<IActionResult> Index(DlsSubApplication dlsSubApplication)
+        {
+            if (!DlsSubApplication.TrackingSystem.Equals(dlsSubApplication) &&
+                !DlsSubApplication.Frameworks.Equals(dlsSubApplication))
+            {
+                return NotFound();
+            }
+
+            var trackingSystemSupportEnabled =

```

Comment:
I don't think these bool names are quite right. Tracking system support might be enabled even if the user is currently trying to access frameworks/resources. The bools really mean shouldContinueToTrackingSystemResources and shouldContinueToFrameworksResources.

Reply:
I just copied them from the SupportController. Do we want to rename them or is it not worth it given they're going to be replaced?

Reply:
Okay, let's leave a todo for 608 here.